blogs/whats_new_1.7/index.md

```diff
@@ -0,0 +1,153 @@
+
+
+## Network Observability CLI enhancements
+
+Network Observability CLI (NetObserv CLI) is a kubectl plug-in that installs a separate Network Observability from the command line and displays a text-based flows table and can also capture packets to a pcap file.  It uses no storage!  When you exit the program, it cleans up and removes itself.
+
+You don't need to install the Network Observability Operator because NetObserv CLI will do it for you.  Just [follow the instructions here](https://docs.openshift.com/container-platform/4.17/observability/network_observability/netobserv_cli/netobserv-cli-install.html) to install NetObserv CLI.  The prerequisite is to have `oc` installed on your computer.
+
+Make sure you have the 1.7 version, which is the GA version.  Enter:
+```
+$ oc netobserv version
+Netobserv CLI version 1.7.0
+```
+
+There were a number of significant changes in NetObserv CLI from the last Network Observability release.  To start, the command line options changed so that when doing packet capture, you now have many of the same options as flow capture, particularly for the eBPF flow filters.  To see these options, enter `oc netobserv help`.  You may need to widen the terminal to view this legibly.
+
+Regardless whether you do flow capture or packet capture, it displays the flows table.  The screen is pretty self-explanatory on what you can do such as filtering data, showing more columns of data, and increasing or decreasing the number of rows.  To exit, press ctrl-c.  It then prompts to save the capture output.  If you respond with 'y', it saves the data with a timestamp filename in the directory **./output/flow/** for flows and **./output/pcap/** for packets.  Figure 9 shows a screenshot of NetObserv CLI after running:
+```
+$ oc netobserv flows
+```
+
+![NetObserv CLI](images/netobserv_cli.png)
+<p style="text-align: center">Figure 9: Network Observability CLI</p>
+
+For packet capture, it's advisable to set a filter as the amount of data will grow very quickly.  By default, there is a maximum byte size of 50 MB and a maximum time limit of 5 minutes before it exits, whichever comes first, although they can be changed with the `--max-bytes` and `--max-time` options respectively.
+
+What's unique about the packet capture is that it stores enriched Kubernetes data in the pcap file!  Here's a screenshot of Wireshark loaded from a pcapng file created by NetObserv CLI (Figure 10).
+```
+oc netobserv packets --protocol=TCP --port=8443
+```
+
+![Wireshark](images/wireshark-pcapng.png)
+<p style="text-align: center">Figure 10: Wireshark displaying pcapng file</p>
+
+The Frame 6 row is selected.  In the Packet Details section below, click the **>** on the first line to open up the "Frame 6" content.  Then click the **>** on "Interface id" to open up the interface details.  You can then see that source IP address 10.129.0.23 in the row selected corresponds to a pod named **prometheus-k8s-0** in the namespace **openshift-monitoring**.
```

Comment:
You can mention my blog here:
https://developers.redhat.com/articles/2024/09/17/network-observability-demand

When I posted it, they asked me to truncate the size because it was too long so if you get the same feedback you can truncate this section and refer to my blog :wink: